scripts/ocr_runner.py

```
@@ -0,0 +1,198 @@
+        return 0
+
+
+def run_ocrmypdf(input_pdf, out_pdf, log_path, timeout):
+    with open(log_path, 'wb') as lf:
+        proc = subprocess.run(['ocrmypdf', '--deskew', '--force', '--output-type', 'pdf', str(input_pdf), str(out_pdf)], stdout=lf, stderr=lf, timeout=timeout)
+    return proc.returncode
+
+
+def fallback_tesseract(input_pdf, out_txt_path):
+    # use pdftoppm -> tesseract
+    tmp_prefix = f'/tmp/{out_txt_path.stem}_page'
```

Ruff (0.14.10)

Probable insecure usage of temporary file or directory: "/tmp/"

(S108)